spec/models/notification_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe Notification, type: :model do
+  let!(:some_notification) { create(:notification) }
+  let!(:super_admin) { create(:user, :super_admin)}
+  let!(:organization) { create(:organization) }
+  let!(:admin) { create(:user, :admin, organization: organization) }
+  let!(:manager) { create(:user, :manager, organization: organization) }
+
+  describe 'factory specs' do
+    it 'has factory' do
+      expect(some_notification).to be_persisted
```

Comment:
another one, factory bot raises error if it unable to create record using `create` helper
useless test